Formula/s/sequin.rb

```diff
@@ -0,0 +1,20 @@
+class Sequin < Formula
+  desc "Human-readable ANSI sequences"
+  homepage "https://github.com/charmbracelet/sequin"
+  url "https://github.com/charmbracelet/sequin/archive/refs/tags/v0.2.0.tar.gz"
+  sha256 "f78cc05bd476ec8e928ab0fda62f9475d63d3c1a9a6c0d229d8eae80202e3fe0"
+  license "MIT"
+  head "https://github.com/charmbracelet/sequin.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -X main.Version=#{version}")
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/sequin -v")
+
+    assert_match "CSI m: Reset style", shell_output("printf '\x1b[m' | #{bin}/sequin")
```

Comment:
```suggestion
    assert_match "CSI m: Reset style", pipe_output(bin/"sequin", "\x1b[m")
```

Reply:
This does not work unfortunately. This is one of upstreams examples:

```
╰─ sequin '\x1b[m'
Error: exec: "\\x1b[m": executable file not found in $PATH
Usage:
  sequin [flags]

Examples:

printf '\x1b[m' | sequin
sequin <file
sequin -- some command to execute


Flags:
  -h, --help      help for sequin
  -r, --raw       raw mode (no explanation)
  -v, --version   version for sequin

exec: "\\x1b[m": executable file not found in $PATH
```

Reply:
No problem! :D